integrations/octopus/src/parser.ts

```diff
@@ -0,0 +1,21 @@
+export const parseBlockByNumber = async (
+  blockNumber: number,
+  api: any,
+  client: any
+): Promise<{ block: any; tags: any }> => {
+  const hash = await api.rpc.chain.getBlockHash(blockNumber);
+
+  const result = await client.request({
+    method: "chain_getBlock",
+    params: [hash.toHex()],
+  });
+  let block = JSON.stringify(result);
+
+  const tags = [
+    { name: "ChaidId", value: 9999 },
```

Comment:
@baidang201 Doesn't look like I have access to push changes to this branch anymore.

Mind committing the following changes? 😄 Then we can get this merged!

```suggestion
    {
      name: "ChainId",
      value: (await api.query.octopusAppchain.appchainId()).toHuman(),
    },
```

Reply:
Welp, looks like GitHub allowed me to make the change 🤷🏻‍♂️